python/tvm/topi/testing/pool3d_python.py

```diff
@@ -20,12 +20,20 @@
 import numpy as np
 import tvm
 
-def pool3d_ncdhw_python(np_data, kernel,
-                        strides, padding,
-                        out_shape, pool_type,
-                        count_include_pad=True,
-                        ceil_mode=False, dtype="float32"):
+
+def pool3d_ncdhw_python(
+    np_data,
+    kernel,
+    strides,
+    padding,
+    out_shape,
+    pool_type,
+    count_include_pad=True,
+    ceil_mode=False,
+    dtype="float32",
+):
     """baseline for max_pool3d and avg_pool3d, default layout is "NCDHW"""
+    # fmt: off
```

Comment:
This looks good, we should probably send an issue to black devs, just haven't gotten around to it.